essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/ZoomRoom/ZoomRoom.cs

```diff
@@ -2511,6 +2511,8 @@ public void LinkZoomRoomToApi(BasicTriList trilist, ZoomRoomJoinMap joinMap)
 
 
 			trilist.SetStringSigAction(joinMap.SubmitPassword.JoinNumber, SubmitPassword);
+			trilist.SetStringSigAction(joinMap.CancelPasswordPrompt.JoinNumber,
+				delegate { OnPasswordRequired(false, false, true, ""); });

```

Comment:
In the future, this should be refactored to `(s) => OnPasswordRequired(false, false, true, "")`